zlib.h.in

```diff
@@ -57,11 +57,11 @@ extern "C" {
 #define ZLIBNG_VER_STATUS 0         /* 0=devel, 1-E=beta, F=Release */
 #define ZLIBNG_VER_MODIFIED 0       /* non-zero if modified externally from zlib-ng */
 
-#define ZLIB_VERSION "1.2.11.zlib-ng"
-#define ZLIB_VERNUM 0x12bf
+#define ZLIB_VERSION "1.2.12.zlib-ng"
+#define ZLIB_VERNUM 0x12c0
```

Comment:
and here, needs to end with an f

Reply:
I have made it also `1.2.12f.zlib-ng`

Reply:
We don't really need the `.f` here since it already says zlib-ng in the string, the .f is a special value so applications can identify it is a modified zlib and not the official one.
Also, it is supposed to be `1.2.12.f`, and not `1.2.12f` 😉

Reply:
Version strings should be decimal, not hexadecimal, except the final part which can be arbitrary string, where generally `a`-`f`, `0`-`9` and `-` are allowed.